### ISSUE TITLE
Please return type 6

### ISSUE THREAD

**Issue by childledj** [youridijk/homebridge-kaku-ics2000#26]
Can you please return type 6 (doorbell button) to devices, it helps me to do some automations

**Comment by childledj** [comment]
Nope, I have real Trust (CoCo or KlikaanKlikuit) doorbells with buttons, that I’ve been used for automations (when door open - doorbells are ringing. It was realized thru turning on doorbell button (it was switch in HK) by door sensor) and after some plugin update buttons disappear from homebridge/homekit.

**Comment by youridijk** [comment]
Ah I understand. I indeed changed this. Previously every non-identified device was shown as a switch. Right now, devices that aren't identified aren't shown. But you can override this in your config.json. To show device type 6 as a switch, you need to add an override for device type 6: 

```json
"deviceConfigOverrides": {
    "6": {
      "modelName": "ACDB 7000A doorbell",
      "onOffFunction": 0
    }
  }
```

**Comment by childledj** [comment]
![Снимок экрана 2023-07-12 в 13 20 22](https://github.com/youridijk/homebridge-kaku-ics2000/assets/38331621/e58b76ee-0d6e-4b08-bb01-c8ac447c7ec3)
Have added but nothing changes after reboot (no buttons)

**Comment by youridijk** [comment]
Sorry, for the late response. Try flushing the cache or re-adding the HomeBridge to HomeKit.

**Comment by childledj** [comment]
Sorry for late answer!
I have tried almost everything:
Added overrides, remove and add ics-2000 plugin
But these for devices did not appear in Homebridge nor Homekit
As for the sensors - they are not necessary, but it would be great if they worked.
As for doorbell chimes - they definitely worked before some update at late 2022 - early 2023.

This is part of my config (is this syntax correct?):
![Снимок экрана 2024-07-27 в 11 53 56](https://github.com/user-attachments/assets/c7f0fb27-f443-4174-b687-c1701423778b)

And this is HB log after boot up
![Снимок экрана 2024-07-27 в 11 51 39](https://github.com/user-attachments/assets/15c4d7f7-8744-48da-bc01-fb38f8604dc2)
